frontend/src/js/components/gmedits/Missing.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+
+function verify(value) {
+    let new_value;
+    if (typeof(value) === 'string') {
+        if(value.match(/[^0-9]+/) || value === '') {
+            return false;
+        } else {
+            new_value = Number.parseInt(value)
+            if (new_value < 0 || new_value > 255) {
+                return false;
+            } else {
+                return new_value;
+            }
+        }
+    } else {
+        console.log("Missing is not a string")
+    }
+}
+var Missing = React.createClass({
+    propTypes: {
+        handleChange: React.PropTypes.func,
+        missing: React.PropTypes.number,
+        className: React.PropTypes.string
+    },
+    getInitialState() {
+        return {
+            missing: this.props.missing,
+        }
+    },
+    componentWillReceiveProps(nextProps) {
+        this.setState({
+            missing: nextProps.missing
+        })
+    },
+    handleBlur(event) {
+        let value = verify(event.target.value);
+        if (value === 0 || value) {
+            this.props.handleChange('missing', value)
+        } else {
+            // indicate user entered wrong value
+            console.log("Missing property must be an integer >= 0 and <=255. Value provided was " + event.target.value)
```

Comment:
This isn't actually indicating that a user entered the wrong value. This is indicating a developer entered the wrong value. Feedback should be provided at the UI level, not on the console.